lnbits/core/crud.py

```diff
@@ -7,7 +7,7 @@
 import shortuuid
 
 from lnbits import bolt11
-from lnbits.db import COCKROACH, POSTGRES, Connection, Filters
+from lnbits.db import COCKROACH, POSTGRES, Connection, Filters, Page
```

Comment:
where is `Page` defined? 
I could not find it.

Reply:
https://github.com/lnbits/lnbits/pull/1613/files#diff-5855a9aec8f89dbcb613cd599d551e090dc55f68b23b20b7c2473261db814dadR111

its inside db.py

Reply:
ah, right
db.py content was hidden in the web view